### ISSUE TITLE
Decimals columns in european de-DE culture misbehaving and null-ing the values

### ISSUE THREAD

**Issue by farcasclaudiu** [mehrgithub/dcrmeg#10]
Decimals columns in european "de-DE" culture misbehaving and null-ing the values.
In (de-DE) culture decimal point is [,] and thousand separator is [.]

**Comment by mehrgithub** [comment]
- Does this issue occur when the grid loads for the first time or when an attempt is made to modify the value in a numeric column?
- Do you see any JavaScript errors in the browser console window?
- What version of CRM is used?

**Comment by farcasclaudiu** [comment]
Ok, apologies, it is our mistake.
We have changed a bit the main code to adapt for our needs, one change was to use <div> elements instead of <span> for editable fields and that failed width jquery .find() .. in the updates.

Just to answer your questions: we use it in a Dynamics 365 online solution, no errors in the browser console, we just modified a [quantity] field and got always nulled because of the mismatch.

Want to point out that your work in this project is awesome and we appreciate it. Thumbs up!
There are a few places for improvements but that's why we started to customize the component to fit our scenario.

Kind regards,
Claudiu
